src/components/SubmissionForm/Confirm.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
If you're not using `useState` and `useEffect`, no need to import them, it'll just clog the browser console with `'useEffect/useState' is defined but never used  no-unused-vars` warnings 🏁

Reply:
This is something I forget all the time too, but I'm trying to watch it like a hawk now 😈